### PR TITLE
doc/user update webhook quickstart

### PR DESCRIPTION
**Pull request by chuck-alt-delete**
### Motivation

Closes https://github.com/MaterializeInc/materialize/issues/25248

### CODE REVIEW

**Comment by chuck-alt-delete** [comment]
We can remove `IN CLUSTER` for the webhook source creation once this hits prod:
- https://github.com/MaterializeInc/materialize/pull/25467

**Comment by def-** [comment]
Something seems to have gone wrong here.

File: doc/user/content/ingest-data/webhook-quickstart.md [hunk 1]
```diff
@@ -64,6 +64,59 @@ to shape the events.
...
+SUBSCRIBE(SELECT * FROM webhook_demo_parsed) WITH (SNAPSHOT = FALSE);
+```
+Press the "Stop Streaming" button when you want to cancel the subscription.
+
+## Step 6. Clean up
+
+Be kind and rewind. Execute this statement to drop all resources created in this tutorial.
+
+```sql
+DROP SECRET demo_webhook CASCADE;
```
Comment (morsapaes) [inline doc/user/content/ingest-data/webhook-quickstart.md:117]:
```suggestion
DROP SOURCE webhook_demo CASCADE;

DROP SECRET demo_webhook;
```

Comment (morsapaes) [reply doc/user/content/ingest-data/webhook-quickstart.md:117]: It's awkward to cleanup using the secret.

**Review by morsapaes — APPROVED** [review]

File: doc/user/content/ingest-data/webhook-quickstart.md [hunk 2]
```diff
@@ -35,8 +35,10 @@ a secure location.
 
 ## Step 2. Set up a webhook source
 
-Using the secret from **Step 1.**, create a webhook source to ingest data from
-the webhook event generator:
+Using the secret from the previous step, create a webhook source to ingest data
+from the webhook event generator. Replace `my_cluster` with the cluster you
```
[the remark this answers is not in the document]
Comment (morsapaes) [reply doc/user/content/ingest-data/webhook-quickstart.md:39]: Oh, I was reviewing this in autopilot. For sure we should omit it.

**Pull request merged**